glory wanted a **not outdated** list (#8)
Anastasia-Romanov/lacan-user-guide: nitrotype.md

- Breaks without manage messages.
- Literally the worst paginator I've ever seen, you can see more detailed information about the paginator [here](/paginator.md).
- Incapable of loading images apparently.

> - Never mind this command doesn't respond with anything anyways in the first place smh

Reviewer: ```suggestion
- Is removed due to it being broken and not responding rather than being fixed.
```

## register

> - input: why require user to type n.verify when you can provide instructions now

Reviewer: ```suggestion
- Input: Why require the user to type `n.verify` when you can provide instructions here.
```

## stats

- Allows you to @ an unverified user and it'll show the stats for whatever account they're pending verification of.  
  - Opens up the possibility for people to mistakenly register to the wrong account and well imagine if they won a giveaway and the host used n.stats to check the account. RIP them.
  - Opens up the ability for impersonation.
  - "Just run n.id username to find out whether the account is verified or not." - glory says, because that is ever-so-user-friendly.  Just fix it for christs sake.

> - time stamps are a thing and we're still using plaintext to show birthday

Reviewer: ```suggestion
- Discord supports timestamps which provide a much better interface but Lacan still uses plaintext for the users account creation.
```

## team

> - Fix the command to where it actually provides team info.

Reviewer: ```suggestion
- Completely broken, responds with "Couldn't find that team" for valid teams.
```
